tf_sig_build_dockerfiles/devel.usertools/gpu.bazelrc

```diff
@@ -41,12 +41,9 @@ build --@local_config_cuda//:enable_cuda
 build --repo_env TF_NEED_CUDA=1
 build --action_env=TF_CUDA_VERSION="11"
 build --action_env=TF_CUDNN_VERSION="8"
-build --action_env=CUDA_TOOLKIT_PATH="/usr/local/cuda-11.2"
+build --action_env=CUDA_TOOLKIT_PATH="/usr/local/cuda"
 build --action_env=GCC_HOST_COMPILER_PATH="/dt9/usr/bin/gcc"
-# TensorRT 7 for CUDA 11.1 is compatible with CUDA 11.2, but requires
-# libnvrtc.so.11.1. See https://github.com/NVIDIA/TensorRT/issues/1064.
-# TODO(b/187962120): Remove CUDA-11.1 when upgrading to TensorRT 8.
-build --action_env=LD_LIBRARY_PATH="/usr/local/cuda:/usr/local/cuda/lib64:/usr/local/cuda/extras/CUPTI/lib64:/usr/local/tensorrt/lib:/usr/local/cuda-11.1/lib64""
+build --action_env=LD_LIBRARY_PATH="/usr/local/cuda/lib64:/usr/local/cuda/extras/CUPTI/lib64"
 build --crosstool_top=@ubuntu20.04-gcc9_manylinux2014-cuda11.2-cudnn8.1-tensorrt7.2_config_cuda//crosstool:toolchain
```

Comment:
It seems that this line should also be updated, but a CUDA 11.7, cudnn 8.4, TRT 8.4 docker image needs to be created first.

Reply:
Right. We need those for remote-build-execution testing, but AFAIK the container should work as-is for building locally. The toolchain in remote_config, if I understand correctly, unnecessarily couples the RBE env variables with the c++ toolchain, which itself respects local env variables. So we could do some non-rbe testing with this as-is, but we'll need to have the crosstool updated eventually.